docs/api/ethereum.md

```diff
@@ -1,5 +1,645 @@
 # API / Ethereum
 
+## Bitski back-end provider
+
+A [Bitski](https://www.bitski.com/) back-end provider is applied to back-end Node.js usage. You will be able to connect to the Ethereum blockchain and create queries and mutations via your Bitski developer wallets. To set up Bitski back-end provider, you need to first create a [development account](https://developer.bitski.com/) on Bitski.
+
+### BitskiProvider(options)
+
+A class providing communication with the Ethereum blockchain through [Bitski](https://www.bitski.com/).
+
+**Arguments**
```

Comment:
This table is broken when rendered on GitHub. This applies also the other bigger tables so please check the rest.